liquigraph-core/src/main/java/org/liquigraph/core/model/Changeset.java

```diff
@@ -147,8 +146,7 @@ public boolean equals(Object obj) {
         }
         final Changeset other = (Changeset) obj;
         return Objects.equals(this.id, other.id) &&
-            Objects.equals(this.author, other.author) &&
-            Objects.equals(this.checksum, other.checksum);
```

Comment:
Please note that I changed changeset equals/hashcode contract: it doesn't take checksum into account anymore.

I didn't see any impact, but I think it's important enough to write this comment just in case I missed something...